### DISCUSSION TITLE
Search issue in MUI table

### DISCUSSION THREAD

**Discussion by LokeshJonwal** [KevinVandy/material-react-table#1171] [General]
When we do search in table as shown below we get the result which is not even matching (see the last row). Is there any way to solve this.

<img width="1587" alt="image" src="https://github.com/KevinVandy/material-react-table/assets/43009495/10ecbc88-9317-428b-bedd-f96de8c3bcd9">

**Comment by KevinVandy** [comment]
Fuzzy search is enabled by default, as described in the docs. Feel free to turn it off for a simpler implementation.
